packages/web-app-files/tests/unit/helpers/resource/resourcesTransfer.spec.ts

```diff
@@ -74,8 +75,12 @@ describe('resourcesTransfer', () => {
     it.each([TransferType.COPY, TransferType.MOVE])(
       'should copy / move files without renaming them if no conflicts exist',
       async (action: TransferType) => {
+        const listFilesResult = {
+          resource: {} as Resource,
+          children: [] as Resource[]
+        } as ListFilesResult
```

Comment:
Why not?
```suggestion
        const listFilesResult: ListFilesResult = {
          resource: {},
          children: []
        }
```

Reply:
resource braucht as a Resource sonst meckert typescript ^^